.travis.yml

```diff
@@ -6,14 +6,17 @@ php:
   - 5.5
   - 5.6
   - hhvm
+  - 7
 
 env:
-  - VARNISH_VERSION=3.0
+  - VARNISH_VERSION=4.0
 
 matrix:
   include:
     - php: 5.5
-      env: VARNISH_VERSION=4.0
+      env: VARNISH_VERSION=3.0
+    - php: 5.3
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
ah thats the problem! i was tinking the php 5.3.3 box had an issue - it has issues with not supporting ssl in php, so i switched to php 5.3 box... but i think you are right.